include/seqan/sequence/string_packed.h

```diff
@@ -585,6 +589,76 @@ resize(
     return front(host(me)).i = new_length;
 }
 
+template <typename TValue, typename THostspec, typename TSize, typename TExpand, typename TValue2>
+inline typename Size< String<TValue, Packed<THostspec> > >::Type
+resize(String<TValue, Packed<THostspec> > & me,
+       TSize new_length,
+       TValue2 const & newValue,
+       Tag<TExpand> tag)
+{
+    typedef String<TValue, Packed<THostspec> > TString;
+    typedef PackedTraits_<TString> TTraits;
+    typedef typename Size<TString>::Type TStringSize;
+    typedef typename TTraits::THostValue THostValue;
+
+    TStringSize old_length = length(me);
+    TStringSize old_host_length = length(host(me));
+
+    if (static_cast<TStringSize>(new_length) < length(me))
+        return resize(me, new_length, tag);
+
+    // create WORD
+    THostValue tmp;
+    tmp.i = 0ul;
+    __uint64 ord = ordValue(TValue(newValue));
```

Comment:
this is a PR against master and develop, so I have to go with something that both understand. I can make an extra PR after that if you want to :smile: